### DISCUSSION TITLE
Applying inferHeterogeneity to ClonEvol

### DISCUSSION THREAD

**Discussion by xiw588** [PoisonAlien/maftools#1064] [General]
**Describe the issue**
Thank you for putting everything together! This is such a great package! I have a quick question on visualizing the results of inferHeterogeneity. I noticed that you also mentioned the package ClonEvol in your tutorial, and wonder if the output from inferHeterogeneity can be used as input for ClonEvol. Thanks!

**Command**
Please post your commands and the output (errors or any unexpected output)

```r
```

**Session info**
Run `sessionInfo()` and post the output below

```r
```

**Comment by PoisonAlien** [comment]
Hi.

Thank you and I am glad you find the package useful.
Regarding the question, unfortunately not. It's only mentioned so that users are aware of similar packages.
